spec/requests/providers/legal_aid_applications_spec.rb

```diff
@@ -4,6 +4,8 @@
   describe 'GET /providers/applications' do
     let!(:legal_aid_application) { create :legal_aid_application }
     subject { get providers_legal_aid_applications_path }
+    let(:provider) { legal_aid_application.provider }
+    let(:other_provider) { create(:provider) }
```

Comment:
No big deal but it would make sense to have `subject` below the 3 `let`s rather than in the middle of them